www/src/data/sidebars/doc-links.yaml

```diff
@@ -764,10 +764,10 @@
     - title: Partnering With Gatsby
       link: /docs/partnering-with-gatsby/
       items:
-        - title: Integrating & Partnering For Vendors
-          link: /docs/gatsby-vendor-partnership/
-        - title: Agency Partnership Program
-          link: /docs/gatsby-agency-partnership/
+        - title: Technology Partner Program
+          link: https://www.gatsbyjs.com/partner-technology
+        - title: Agency Partner Program
+          link: https://www.gatsbyjs.com/partner-agency
```

Comment:
This used to link to the page you edited, Agency Partnership Program. Now nothing links to that page. Is that intentional? Do you also need to edit the page on .com? That will require a PR to [mansion](https://github.com/gatsby-inc/mansion). If we're no longer linking to these two existing pages, the Markdown files should be deleted.

Reply:
Good point. As long as nothing else links to those 'old' pages, I think we can delete them as the new links will direct them to .com. The .com pages are already updated. Also, what are the SEO implications of deleting the old pages? Is it better to keep them live and just redirect to the new partner program page?

Reply:
I'd say do a permanent redirect to the new pages. I can take care of this if everyone agrees.